core/src/App.html

```diff
@@ -87,7 +107,7 @@
     return paths.includes(removeQueryParams(routePath));
   };
 
-  const enableRouting = (component, node, config) => {
+  const enableRouting = (component, node, config, callback) => {
```

Comment:
Maybe it is overkill to pass the callback and we could just execute the callback code inline?

Reply:
done